crypto/threads_pthread.c

```diff
@@ -548,7 +549,7 @@ CRYPTO_RCU_LOCK *ossl_rcu_lock_new(int num_writers, OSSL_LIB_CTX *ctx)
     /* By default our first writer is already alloced */
     new->writers_alloced = 1;
 
-    new->qp_group = allocate_new_qp_group(new, num_writers);
+    new->qp_group = allocate_new_qp_group(new, (uint32_t)num_writers);
```

Comment:
I dont think this type cast is necessary.  The implicit conversion does not overflow.